### ISSUE TITLE
Implement the 24 months plan in the Website

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#5271]
See Epic description for full details. 

In brief:

* retire 6-month plan, add 24-month plan
* [https://www.figma.com/file/kpz1dqP2terBMIZLjiOhev/Free-Trials?node-id=426%3A440](https://www.figma.com/file/kpz1dqP2terBMIZLjiOhev/Free-Trials?node-id=426%3A440|smart-link) shows how our UX team has determined how it should show up on a phone.

Bedrock Team ticket: [https://github.com/mozilla/bedrock/issues/11881](https://github.com/mozilla/bedrock/issues/11881|smart-link)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-3535)
┆Reporter: Sarah Bird

**Comment by data-sync-user** [comment]
➤ Alex Gibson commented:

On bedrock’s side I’ll need:

  * New 24 month plan IDs for each country / language we support.
  * New monthly price (per currency) for the 24 month plan e.g. “US$x.xx/month”.
  * New savings calculation (per currency) e.g. “Save 50%”.
  * New total price (per currency) e.e. “US$xx.xx total”.

In terms of timing, we’ll also require new strings for L10n with this update, so I’ll suggest we should add the new plan behind a feature flag that’s initially only enabled on https://www-dev.allizom.org/products/vpn/. ( https://www-dev.allizom.org/products/vpn/.|smart-link ) 

This will give time for both QA and our translators to do their work prior to the launch date (TBD).

**Comment by data-sync-user** [comment]
➤ Betty Fleming commented:

Waiting on pricing info Tony Cinotto Santiago Andrigo
